src/anyvar/restapi/globals.py

```diff
@@ -5,40 +5,68 @@
 """
 
 import os
+from typing import Any, Callable, Optional
 
 from flask import current_app
-
 from ga4gh.vrs.dataproxy import create_dataproxy, _DataProxy
+from anyvar.translate.translate import TranslatorSetupException
+
+from anyvar.translate.variation_normalizer import VariationNormalizerRestTranslator
 
 from ..anyvar import AnyVar
 from ..storage import create_storage
+from ..translate import _Translator
 
 anyvar_db_fn = os.path.expanduser("/tmp/anyvar.db")
 
 
+def _get_g(key: str, creator_function: Callable) -> Any:
+    """Fetch a global singleton, creating with `fn` on first invocation
 
-def _get_g(k, fn):
-    """fetch a global singleton, creating with `fn` on first invocation"""
-    v = getattr(current_app, k, None)
-    if not v:
-        v = fn()
-        setattr(current_app, k, v)
-    return v
+    :param key: object key associated with current Flask instance
+    :param creator_function: called to create object if it doesn't already exist
+    """
+    singleton_object = getattr(current_app, key, None)
+    if not singleton_object:
+        singleton_object = creator_function()
+        setattr(current_app, key, singleton_object)
+    return singleton_object
 
 
 def get_dataproxy() -> _DataProxy:
     return _get_g("_dataproxy", create_dataproxy)
 
 
-def _create_anyvar():
+def create_translator(uri: Optional[str] = None) -> _Translator:
+    """Create variation translator middleware.
+
+    Currently accepts REST interface only.
+
+    :param uri: location listening for REST requests
+    :return: instantiated Translator instance
+    """
+
+    if not uri:
+        uri = os.environ.get("ANYVAR_VARIATION_NORMALIZER_URI")
+        if not uri:
+            raise TranslatorSetupException(
+                "No Translator object or URI provided."
```

Comment:
Translator object does not get passed 
```suggestion
                "No URI provided."
```